mne/viz/tests/test_topomap.py

```diff
@@ -136,6 +140,19 @@ def test_plot_topomap_animation():
     anim._func(1)  # _animate has to be tested separately on 'Agg' backend.
     plt.close('all')
 
+    fname_nirx_15_0 = op.join(data_path(download=False),
+                              'NIRx', 'nirx_15_0_recording')
+    raw = read_raw_nirx(fname_nirx_15_0, preload=True)
+    raw = nirs.optical_density(raw)
+    raw = nirs.beer_lambert_law(raw)
+    events, _ = events_from_annotations(raw, event_id={'1.0': 1, '2.0': 2})
+    event_dict = {'A/b': 2, 'A/c': 1}
+    epochs = Epochs(raw, events, event_id=event_dict, tmin=-1, tmax=2)
+    evoked = epochs['A'].average()
+    fig, anim = evoked.animate_topomap(ch_type='hbo')
+    anim._func(1)  # _animate has to be tested separately on 'Agg' backend.
+    assert len(fig.axes) == 2
```

Comment:
Originally I tried to change the file types of the evoked variable to hbo for this test. But due to a projection and average referencing I was unable to use the same approach as in my previous PR (see below). This code seems excessive, can anyone suggest a nicer way of testing this?

https://github.com/mne-tools/mne-python/pull/7001/files#diff-95830e06fd39b594b34b28b6fecf7296R424-R428

Reply:
just create fake data with EvokedArray and create_info + set_montage. clear?

Reply:
Thanks @agramfort for the point in the right direction. I am now happy with this PR, could you please review.